subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy

```diff
@@ -16,33 +16,90 @@
 
 package org.gradle.smoketests
 
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
 import org.gradle.util.Requires
 import org.gradle.util.TestPrecondition
+import spock.lang.IgnoreIf
 import spock.lang.Unroll
 
+@Requires(TestPrecondition.ONLINE)
 class BuildScanPluginSmokeTest extends AbstractSmokeTest {
-    @Requires(TestPrecondition.ONLINE)
+
+    private static final List<String> GRACEFULLY_UNSUPPORTED = [
+        "1.6",
+        "1.7",
+        "1.7.1",
+    ]
+
+    private static final List<String> SUPPORTED = [
+        // "latest.integration", - needs to be updated
+    ]
+
     @Unroll
-    def "can run build with build scan plugin #version"() {
-        def initScript = file("init.gradle") << """
-initscript {
-    repositories {
-        maven { url "https://plugins.gradle.org/m2" }
+    @IgnoreIf({ BuildScanPluginSmokeTest.SUPPORTED.empty })
+    "can run build with build scan plugin #version"() {
+        when:
+        usePluginVersion version
+
+        then:
+        with(build()) {
+            output.contains("Publishing build information")
+            output.contains("https://gradle.com/s/")
+        }
+
+        where:
+        version << SUPPORTED
     }
-    dependencies {
-        classpath "com.gradle:build-scan-plugin:${version}"
+
+    @Unroll
+    "gracefully fails with unsupported version #version"() {
+        when:
+        usePluginVersion version
+
+        then:
+        with(buildAndFail("--scan")) {
+            output.contains("""
+> Failed to apply plugin [id 'com.gradle.build-scan']
+   > This version of Gradle requires version 1.7.2 of the build scan plugin or later.
+     Please see https://gradle.com/scans/help/gradle-incompatible-plugin-version for more information.
+""")
+        }
+
+        where:
+        version << GRACEFULLY_UNSUPPORTED
     }
-}
-rootProject {
-    apply plugin: com.gradle.scan.plugin.BuildScanPlugin
-    buildScan {
-          licenseAgreementUrl = 'https://gradle.com/terms-of-service'
-          licenseAgree = 'yes'
+
```

Comment:
I don't think it's worth it. We effectively make no commitment to them working in any fashion.